templates/example.yaml

```diff
@@ -0,0 +1,4 @@
+basic:
+  username1:
+    ID: "orgid"
+    password: "password1"
```

Comment:
you can put this example.yaml in a testdata directory, and use it for a unit test.
https://dave.cheney.net/2016/05/10/test-fixtures-in-go